flatcontent/tests/test_flatcontent.py

```diff
@@ -72,14 +72,57 @@ def test_template_tag(self):
         resp = self.client.get(reverse('template_tag'))
         self.assertContains(resp, 'test content')
 
+    def test_template_tag_as(self):
+        resp = self.client.get(reverse('template_tag_as'))
+
+        self.assertContains(resp, '<p>test content</p>')
+        self.assertContains(resp, 'with first test content')
+        self.assertContains(resp, 'with last test content')
+
     def test_template_tag_with_site(self):
         resp = self.client.get(reverse('template_tag_with_site'))
         self.assertContains(resp, 'test content with site')
 
+    def test_template_tag_with_extra_ctx(self):
+        FlatContentFactory(
+            slug='test-content-with-extra-ctx',
+            site=None,
+            content='test content with {{ var1 }} {{ var2 }}',
+        )
+
+        resp = self.client.get(reverse('template_tag_with_extra_ctx'))
+
+        self.assertContains(resp, 'test content with extra ctx')
+
+    def test_missing_as(self):
+        with self.assertRaises(TemplateSyntaxError) as tse:
+            self.client.get(reverse('missing_as'))
+
+        self.assertEqual(
+            tse.exception.args[0],
+            "The argument after 'as' should be a name for the context var",
```

Comment:
@brad Can we use `FlatContentNode.errors` instead of hardcoding the error strings in the tests? It makes it easier to change the error strings later if we want to do that.